wolfcrypt/src/asn.c

```diff
@@ -9697,7 +9703,9 @@ static int DecodeBasicOcspResponse(byte* source, word32* ioIndex,
             return ASN_PARSE_E;
 
         InitDecodedCert(&cert, resp->cert, resp->certSz, heap);
-        ret = ParseCertRelative(&cert, CERT_TYPE, VERIFY, cm);
+        /* Don't verify if we don't have access to Cert Manager. */
+        ret = ParseCertRelative(&cert, CERT_TYPE,
```

Comment:
Do you understand what I'm asking?  We CANNOT allow simple programming mistakes to accidentally turn off cert verification.  Please add an EXPLICIT parameter to prevent this type of mistake for this function.